src/provider/camunda-platform/properties/ListenerProps.js

```diff
@@ -195,6 +264,18 @@ function EventType({ id, element, listener }) {
   }
 
   function getOptions() {
+
+    if (listener.$type == 'camunda:TaskListener') {
```

Comment:
```suggestion
    if (is(listener, 'camunda:TaskListener')) {
```

We don't want to depend on internals.